src/components/OpenwbBaseButtonGroupInput.vue

```diff
@@ -84,18 +72,41 @@ export default {
         this.$emit("update:modelValue", newValue);
       },
     },
+    useMultiRow() {
+      return this.maxButtonsPerRow && this.buttons.length > this.maxButtonsPerRow;
+    },
+    buttonRows() {
+      if (!this.useMultiRow) return [this.buttons];
+      const rows = [];
+      for (let i = 0; i < this.buttons.length; i += this.maxButtonsPerRow) {
+        rows.push(this.buttons.slice(i, i + this.maxButtonsPerRow));
+      }
+      return rows;
+    },
```

Comment:
Der Datentyp sollte einheitlich sein. So wird auch bei nur einer Reihe ein verschachtelter Array erzeugt und kann einheitlich im Template verarbeitet werden.
```suggestion
    buttonRows() {
      const rows = [];
      for (let i = 0; i < this.buttons.length; i += this.maxButtonsPerRow) {
        rows.push(this.buttons.slice(i, i + this.maxButtonsPerRow));
      }
      return rows;
    },
```

Reply:
ein einheitlicher Datentyp vereinfacht die Verarbeitung im Template.
Ich habe die alte Logik entfernt und buttonRows() so angepasst, dass immer ein Array von Arrays erzeugt wird, unabhängig von der Zeilenanzahl.